modules/Core/src/main/java/org/terasology/logic/health/HealthComponent.java

```diff
@@ -34,7 +34,7 @@
     @Replicate
     public float fallingDamageSpeedThreshold = 20;
     @Replicate
-    public float horizontalDamageSpeedThreshold = 100;
+    public float horizontalDamageSpeedThreshold = 20;
```

Comment:
I think we should have same speed damage in all directions. That's why I changes horizontalSpeed damage to 20. and I think we should just have 1 constant i.e. damageSpeedThreshold that works for all directions.